compose.local.yml

```diff
@@ -5,7 +5,6 @@ services:
     healthcheck:
       interval: 10s
       retries: 10
-      test: ['CMD-SHELL', 'pg_isready -d test -U $${POSTGRES_USER}']
```

Comment:
@alexkutsan why it was removed?

Reply:
Because it does not work. 
it continuously reports an error that the test database does not exist. 
I have tried to change to `bc` database, but it did not work either.

Reply:
I understand how those log messages can be confusing. However, rather than removing the healthcheck entirely in unrelated PR, it would be more appropriate to adjust the healthcheck and/or update the setup instructions.
https://github.com/NeuraLegion/brokencrystals/pull/537